cocos/renderer/gfx/DeviceGraphics.cpp

```diff
@@ -409,6 +409,15 @@ void DeviceGraphics::draw(size_t base, GLsizei count)
     }
     
     commitTextures();
+
+    if(!nextIndexBuffer->isDrawable()){
```

Comment:
May be can check it as early as possible. For example, add after `auto nextIndexBuffer = _nextState->getIndexBuffer();`.